instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Post from './Post';
+import './Post.css';
+
+const PostContainer = props => {
+        return (
+            <div className="post-container-space">
+
+                {props.posts.map(p => <Post key={p.imageUrl} post={p} />)}
```

Comment:
Nice use of the image url as the component key. You also have timestamp available with is a great variable for a key.